app/views/conferences/_schedule_splashpage.html.haml

```diff
@@ -10,13 +10,22 @@
         - if @conference.splashpage and @conference.program.tracks.any? and @conference.splashpage.include_tracks
           See rock-star speakers cover the topics of
       - if @conference.splashpage and @conference.splashpage.include_tracks
-        - @conference.program.tracks.each_slice(3) do |slice|
+        - @conference.program.tracks.confirmed.cfp_active.each_slice(3) do |slice|
```

Comment:
What if the cfp_active attribute is false for a self_organized track, because its cfp ended sooner?